.github/workflows/AWS-auth.yml

```diff
@@ -0,0 +1,68 @@
+name: AWS Auth + Welcome Email
+
+on:
+  push:
+    branches:
+      - 'test_userpool_creation'
+    paths:
+        '**.json'
+
+env:
+  AWS_ACCT_ID: ${{  secrets.AWS_ACCT_ID  }}
+  AWS_REGION : 'us-west-2'
+  IAM_ROLE: ${{  secrets.ROLE_NAME  }}
+
+permissions:
+      id-token: write
+      contents: read
+jobs:
+  changed_files:
+    runs-on: ubuntu-latest  # windows-latest || macos-latest
+    name: Get config file name
+    outputs:
+      config-file-name: ${{ steps.config-file-name.outputs.CONFIG_FILE}}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # OR "2" -> To retrieve the preceding commit.
+
+      - name: Get changed files
+        id: get-changed-files
+        uses: tj-actions/changed-files@v40
+
+      # NOTE: `since_last_remote_commit: true` is implied by default and falls back to the previous local commit.
+
+      - name: List all changed files
+        id: config-file-name
+        run: |
+          for file in ${{ steps.get-changed-files.outputs.all_changed_files }}; do
+            if [[ "$file" == *nrel-op.json ]]; then
+              echo "The name of the config file is: ${file}."
+              echo "CONFIG_FILE=${file}" >> "$GITHUB_OUTPUT"
+            fi
+          done
```

Comment:
what happens if there were multiple changed files? This looks like it will return only the last value. Won't we then skip the modifications in the other changed files?